src/screens/Cart.tsx

```diff
@@ -157,24 +157,34 @@ export default function Cart() {
             </Text>
           </Row>
         </Box>
-        <Box mt={5} bg='white' px={4} py={6} width={'100%'} flexWrap={'wrap'}>
+        <Box
+          mt={5}
+          bg='white'
+          px={4}
+          py={6}
+          width={window.width}
+          flexWrap={'wrap'}
+        >
           <Box
             bg='orange.50'
             p={4}
             borderRadius={8}
             borderWidth={1.5}
             borderColor='orange.300'
             flexDirection={'row'}
+            width={window.width - 30}
           >
-            <Checkbox
-              colorScheme='orange'
-              borderColor='orange.700'
-              mr={3}
-              value={''}
-            >
-              {''}
-            </Checkbox>
-            <Box width='90%' color='orange.700'>
+            <Box width={42}>
+              <Checkbox
+                colorScheme='orange'
+                borderColor='orange.700'
+                mr={3}
+                value={''}
+              >
+                {''}
```

Comment:
Why are we adding this?

Reply:
It was giving me Warning for line no 184.
I have removed this line and added **accessibilityLabel props in checkbox**
### WARNING. **If you do not provide children, you must specify an accessibilityLabel for accessibility**

For Line 182  _**value={''}**_, Eslint error is coming 
**Property 'value' is missing in checkbox**